bin/git-ignore

```diff
@@ -11,18 +11,30 @@ function show_contents {
   fi
 }
 
+function maybe_cd_to_git_root {
+  local result=
+  if result=$(git rev-parse --show-toplevel 2>/dev/null); then
```

Comment:
Let's add this cmd in https://github.com/tj/git-extras/blob/master/not_need_git_repo